src/transport/AdminPairingTable.cpp

```diff
@@ -258,15 +307,24 @@ CHIP_ERROR AdminPairingInfo::SetOperationalCert(const ByteSpan & cert)
         ReleaseOperationalCert();
     }
 
+    ByteSpan noc;
+    ByteSpan icac;
+    ConvertChipCertArrayToChipCerts(cert, noc, icac);
```

Comment:
The overall method `SetOperationalCert` should be renamed `SetOperationalCertArray`.

Reply:
Will update.